flink-sql-examples/kafka-bq.sql

```diff
@@ -0,0 +1,37 @@
+-- Reads from Managed Service for Apache Kafka and writes to BigQuery.
+
+-- Replace values in <ANGLE BRACKETS>
+CREATE TABLE kafka_source (
+    word STRING,
+    proctime AS PROCTIME() -- Generate processing-time attribute using computed column.
+) WITH (
+    'connector' = 'kafka',
+    'topic' = '<TOPIC>',
+    'scan.startup.mode' = 'earliest-offset',  -- Read from the beginning.
+    'properties.bootstrap.servers' = 'bootstrap.<CLUSTER>.us-central1.managedkafka.<PROJECT>.cloud.goog:9092',
```

Comment:
we probably want to replace the <LOCATION> for gmk clusters in different regions

Reply:
makes sense - fixed